### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
This one is really big. I would suggest you guys to skip documentation.html file as that one is too large. Prism css and js files are for displaying code snippets so ignore those files. 

Rest are small codes changes that I have refactored and added some new utilities so do check the rest of files.
I have cross checked documentation.html file, but if you think you can review go ahead. But keep it last as its really huge.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -2,42 +2,267 @@
...
-}
-
-.container{
+.container {
     max-width: 80rem;
-    margin:auto;
+    margin: auto;
     padding: 1.6rem;
     min-height: 100vh;
     background-color: #eee;
```
Comment (zoef-yt) [inline style.css]: use var for colour.

File: style.css [hunk 2]
```diff
@@ -2,42 +2,267 @@
...
     flex-wrap: nowrap;
 }
 
-.flex-stretch{
-    align-items: stretch;
+/* LANDING PAGE */
+
+.nav {
+    display: flex;
+    /* background-color: var(--PRIMARY-50); */
```
Comment (zoef-yt) [inline style.css]: remove commented code

File: style.css [hunk 3]
```diff
@@ -2,42 +2,267 @@
...
+}
+
+.hero-button-secondary {
+    padding: 1.5rem 2rem;
+    font-size: 2.2rem;
+    border: none;
+    background-color: transparent;
+    border-radius: 5px;
+    border: 2px solid var(--OFF-BLACK);
+    /* color: var(--OFF-WHITE); */
```
Comment (zoef-yt) [inline style.css]: remove commented code

File: style.css [hunk 4]
```diff
@@ -2,42 +2,267 @@
...
+    gap: 2rem;
+}
+
+.hero-button-primary {
+    padding: 1.5rem 2rem;
+    font-size: 2.2rem;
+    border: none;
+    background-color: #4be190;
+    border-radius: 5px;
+    /* color: var(--OFF-WHITE); */
```
Comment (zoef-yt) [inline style.css]: remove commented code

File: style.css [hunk 5]
```diff
@@ -2,42 +2,267 @@
...
+    position: sticky;
+    top: 0;
+    grid-area: navbar;
+    z-index: 9;
+}
+
+.doc-sidebar {
+    grid-area: sidebar;
+    /* position: fixed;
+    top: 6rem; */
```
Comment (zoef-yt) [inline style.css]: remove commented code

File: style.css [hunk 6]
```diff
@@ -2,42 +2,267 @@
...
+    overflow-y: scroll;
+}
+
+.doc-sidebar::-webkit-scrollbar{
+    display: none;
+}
+/* 
+.doc-sidebar:hover::-webkit-scrollbar{
+    display: block;
+} */
```
Comment (zoef-yt) [inline style.css]: remove commented code

File: components/utilities/text-utilities/text-utilities.css [hunk 7]
```diff
@@ -8,6 +8,7 @@
 .h1,
 h1 {
     font-size: 3.2rem;
+    /* font-weight: 700; */
```
Comment (zoef-yt) [inline components/utilities/text-utilities/text-utilities.css]: remove commented code

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: style.css (hunk 1 above)
Comment (bhushannaikaroskar) [reply style.css]: oh that container class was extra code will remove

**Review by zoef-yt — APPROVED** [review]
The code looks good. Suggestion adding some screenshots of the changes you made in the UI would be great. Good to merge.

File: app.js [hunk 8]
```diff
@@ -0,0 +1,24 @@
+
+
+const componentList = document.querySelector("#sidebar-list");
+const arrayOfListItems = componentList.querySelectorAll(".list-item");
+const docMain = document.querySelector(".doc-main");
+
+
+const activeTab = (list,listItem) =>{
+    const arrayOfTabs = list.querySelectorAll(".list-item");
+    console.log("entering")
```
Comment (iRohitGaur) [inline app.js]: remove console.log

File: components/avatar/avatar.css [hunk 9]
```diff
@@ -60,7 +60,7 @@
 .avatar-badge{
     position:absolute;
     box-sizing:content-box ;
-    background-color:green;
+    background-color: var(--PRIMARY-300);
     border-radius: 999px;
     border: 3px solid var(--COLOR-BORDER);
     width:20%;
```
Comment (iRohitGaur) [inline components/avatar/avatar.css]: spacing not consistent

File: components/utilities/sizing/sizing.css [hunk 10]
```diff
@@ -33,3 +33,99 @@
 .h-100 {
     height: 100%;
 }
+
+
+/* PADDING */
+
+
+.p-0{
+    padding: 0rem;
```
Comment (iRohitGaur) [inline components/utilities/sizing/sizing.css]: no need of unit. use 0

File: style.css [hunk 11]
```diff
@@ -2,42 +2,241 @@
...
+.button-container {
+    display: flex;
+    gap: 2rem;
+}
+
+.hero-button-primary {
+    padding: 1.5rem 2rem;
+    font-size: 2.2rem;
+    border: none;
+    background-color: #4be190;
```
Comment (iRohitGaur) [inline style.css]: use var

File: style.css [hunk 12]
```diff
@@ -2,42 +2,241 @@
...
+    border-radius: 5px;
+    box-shadow: 0 2px 10px 2px #00d97755;
```
Comment (iRohitGaur) [inline style.css]: use var

File: style.css [hunk 13]
```diff
@@ -2,42 +2,241 @@
...
+.doc-body {
+    display: grid;
+    grid-template-columns: 0 auto 1fr 0;
+    grid-template-rows: auto 1fr;
+    grid-template-areas: 
+            "navbar navbar navbar navbar"
+            "... sidebar main ..." ;
+    background-color: var(--COLOR-SHADOW-PRIMARY-50);
+    min-height: 100vh;
+    max-height: 100vh;
```
Comment (iRohitGaur) [inline style.css]: use height here

File: style.css [hunk 14]
```diff
@@ -2,42 +2,241 @@
...
+}
+
+.doc-main {
+    grid-area: main;
+    padding: 2rem;
+    overflow-y: scroll;
+}
+
+.doc-header{
+    margin: 0rem 0  2rem 0;
```
Comment (iRohitGaur) [inline style.css]: update this: margin: 0 0  2rem;

**Review by iRohitGaur — CHANGES_REQUESTED** [review]
Please do the updates as discussed in the VC

**Review by iRohitGaur — APPROVED** [review]
Good to merge with updates 👍🏻

**Pull request merged**